cmake/TenzirRegisterPlugin.cmake

```diff
@@ -520,6 +520,7 @@ function (TenzirRegisterPlugin)
   file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/stub.h" "")
   list(APPEND PLUGIN_SOURCES "${CMAKE_CURRENT_BINARY_DIR}/stub.h")
   add_library(${PLUGIN_TARGET} OBJECT ${PLUGIN_SOURCES})
+  set_target_properties(${PLUGIN_TARGET} PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
Out of curiosity: Why would this be needed?

Reply:
CMake wasn't able to deduce this automatically and failed to configure builds. Usually this happens when there are no sources to extract this info from but the plugins had sources. So I just hot-fixed this here.

Reply:
Please leave a comment right above the line explaining how that could happen.

Reply:
Added, please check.

Reply:
I think that works. Thanks!